contentreports-filteredcollections.md

```diff
@@ -0,0 +1,149 @@
+
+## Statistics for the whole repository
+**GET /api/contentreports/filteredcollections**
+
+**POST /api/contentreports/filteredcollections**
+
+This endpoint provides aggregated statistics about the number of items per collection according to selected filters.
+
+For each collection, the basic report consists of:
+* name (label) and handle of the collection
+* name (label) and handle of the parent community
+* total number of items
+* number of items matching all selected filters
+
+In addition, a `summary` element provides the total number of items and the total number of items matching all filters
+for the whole repository.
+
+An example JSON response document to `/api/contentreports/filteredcollections`:
+```json
+{
+    "id": "filteredcollections",
+    "collections": [
+        {
+            "label": "Collection 1",
+            "handle": "100/1",
+            "values": {
+                "is_discoverable": 23,
+                "has_multiple_originals": 3,
+                "has_pdf_original": 14
+            },
+            "community_label": "Community 1",
+            "community_handle": "20.500.11794/1",
+            "nb_total_items": 23,
+            "all_filters_value": 3
+        },
+        {
+            "label": "Collection 2",
+            "handle": "100/2",
+            "values": {
+                "is_discoverable": 1,
+                "has_multiple_originals": 0,
+                "has_pdf_original": 0
+            },
+            "community_label": "Community 1",
+            "community_handle": "20.500.11794/1",
+            "nb_total_items": 1,
+            "all_filters_value": 0
+        },
+        {
+            "label": "Collection 3",
+            "handle": "100/3",
+            "values": {
+                "is_discoverable": 1,
+                "has_multiple_originals": 0,
+                "has_pdf_original": 1
+            },
+            "community_label": "Community 1",
+            "community_handle": "20.500.11794/1",
+            "nb_total_items": 1,
+            "all_filters_value": 0
+        }
+    ],
+    "summary": {
+        "label": null,
+        "handle": null,
+        "values": {
+            "is_discoverable": 25,
+            "has_multiple_originals": 3,
+            "has_pdf_original": 15
+        },
+        "community_label": null,
+        "community_handle": null,
+        "nb_total_items": 25,
+        "all_filters_value": 3
+    },
+    "type": "filtered-collections",
+    "_links": {
+        "self": {
+            "href": "http://localhost:8080/dspace-server/api/contentreports/filtered-collections"
+        }
+    }
+}
+```
+
+The request can be parameterized with a series of filters to add to the basic report.
+
+In GET mode, it consists of a `filters` query parameter whose value is a comma-separated list of filters
+like the following:
+```
+?filters=is_discoverable,has_multiple_originals,has_pdf_original
+```
+
+In POST mode, it is defined as a JSON document like this:
```

Comment:
I'd prefer if we describe GET and POST mode separately.  I'm having a hard time understanding the way this is documented. When would someone use GET and when would they use POST?  It's unclear if everything below this point in the docs is ONLY for POST or if it also applies to GET?  Could we give some more examples here as to what the differences are?

Reply:
I reorganized report parameterization in the Filtered Collections report documentation. I also fixed a few mistakes and added some info I realised that was missing (e.g., definition of "basic report" in Filtered Items).

About usage of GET vs. POST, please see my other comments below.